client/modules/_hooks/src/datafiles/projects/types.ts

```diff
@@ -21,7 +21,7 @@ export type TReferencedWork = {
 };
 
 export type TAssociatedProject = {
-  type: 'Context' | 'Linked Dataset' | 'Cited By';
+  type: 'Context' | 'Linked Dataset or Software' | 'Cited By';
```

Comment:
This change can be reverted as well since the JSON from the backend will still have the `'Linked Dataset'` wording.